package.json

```diff
@@ -17,16 +17,16 @@
     "url": "https://github.com/qlik-oss/sn-table.git"
   },
   "files": [
-    "assets",
-    "dist"
+    "dist",
+    "api-specifications"
   ],
   "engines": {
     "node": ">=8"
   },
   "main": "dist/sn-table.js",
   "module": "dist/sn-table.esm.js",
   "scripts": {
-    "build": "rollup -c rollup.config.js && shx cp assets/* dist",
+    "build": "rollup -c rollup.config.js && rollup -c rollup-core.config.js && shx cp assets/* dist",
```

Comment:
Would be nice to have like a `dev-build` and a `prod-build` so that we don't have to run all this every build